include/psa/crypto_se_driver.h

```diff
@@ -109,15 +186,15 @@ typedef psa_status_t (*psa_drv_se_mac_update_t)(void *p_context,
  * \retval PSA_SUCCESS
  *          Success.
  */
-typedef psa_status_t (*psa_drv_se_mac_finish_t)(void *p_context,
+typedef psa_status_t (*psa_drv_se_mac_finish_t)(void *op_context,
                                                 uint8_t *p_mac,
                                                 size_t mac_size,
                                                 size_t *p_mac_length);
 
 /** \brief A function that completes a previously started secure element MAC
  * operation by comparing the resulting MAC against a provided value
  *
- * \param[in,out] p_context A hardware-specific structure for the previously
+ * \param[in,out] op_context    A hardware-specific structure for the previously
```

Comment:
I'll do a pass of aligning, because that's the general rule in these files. But in the medium term I think we should stop aligning because it has a high maintenance cost and it hurts readability. It causes all the text to be compressed into a very narrow column.